android/app/src/main/java/com/reactnativeapp/MainApplication.java

```diff
@@ -19,9 +21,10 @@
 import java.util.Arrays;
 import java.util.List;
 
-public class MainApplication extends Application implements ReactApplication, ResourceStringCallback {
+public class MainApplication extends Application implements ReactApplication, ResourceStringCallback, CurrencyChangeCallback {
 
     RNTranslationManagerPackage translationPackage = new RNTranslationManagerPackage(this);
+    RNCurrencyManagerPackage currencyPackage = new RNCurrencyManagerPackage(this);
```

Comment:
Why do we have to do it like this?

Reply:
Because the native module we get is just a mock, we have to add our actual own implementation 🥇